app/locker/repository/Statement/EloquentStorer.php

```diff
@@ -41,7 +43,14 @@ public function store(array $statements, array $attachments, StoreOptions $opts)
     $this->voider->voidStatements($statements, $opts);
     $this->attacher->store($attachments, $this->hashes, $opts);
     $this->activateStatements($ids, $opts);
-
+    // Check if there's any webhook set up to queue up WebhookAsyncReq job
+    $lrsId = (string) $opts->getOpt('lrs_id');
+    $webhook = Webhook::where('lrs_id', $lrsId)->first();
```

Comment:
@ht2 please confirm that this is ok (converts a MongoId to a String for filtering).